[19.0][MIG]: sale_stock_picking_invoice_link

Alexgars73 opened the pull request:
Migration to v19

renatonlima committed:
[NEW] sale_stock_picking_invoice_link

Commit:
[UPD] Update sale_stock_picking_invoice_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: stock-logistics-workflow-18.0/stock-logistics-workflow-18.0-sale_stock_picking_invoice_link
Translate-URL: https://translation.odoo-community.org/projects/stock-logistics-workflow-18-0/stock-logistics-workflow-18-0-sale_stock_picking_invoice_link/it/

Alexgars73 committed:
[IMP] sale_stock_picking_invoice_link: pre-commit auto fixes

Alexgars73 committed:
[MIG] sale_stock_picking_invoice_link: Migration to 19.0

mav-adhoc commented:
@Alexgars73 Hi! Can you check the tests? Thanks!

Alexgars73 commented:
Hello! @mav-adhoc What exactly are you referring to? Thanks

mav-adhoc commented:
@Alexgars73 It failed, but it is because it doesn't have the dependencies migrated. Sorry! 
<img width="1019" height="133" alt="image" src="https://github.com/user-attachments/assets/ea006d14-fdb4-422d-9c4a-ee9db9aab254" />

Reyes4711-S73 reviewed (approved):
LGTM

sbiosca-s73 reviewed (approved):
LGTM!

Reyes4711-S73 commented:
@OCA/logistics-maintainers can you merge this PR?

david-s73 committed:
[FIX] sale_stock_picking_invoice_link: Create invoices without all delivery notes

It had to be modified to check that stock.move has to link to each line of the invoice, because if only the ‘to_refund’ field was checked, it did not work correctly because in ODOO it has been set to always be true by default, so it was taking all the lines because it was not subtracting correctly and the ‘to_invoice’ variable always had a value and never reached 0.

hparfr commented:
/ocabot migration sale_stock_picking_invoice_link

hparfr reviewed (approved):
/ocabot merge nobump
